wasmd/cmd/wasmd_init.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/ignite/apps/wasmd/services/scaffolder"
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/pkg/placeholder"
+	"github.com/spf13/cobra"
+)
+
+func NewWasmdInit() *cobra.Command {
+	c := &cobra.Command{
+		Use:     "init",
+		Short:   "Import the wasm module to your app",
+		Long:    "Add support for WebAssembly smart contracts to your blockchain",
+		Args:    cobra.NoArgs,
+		PreRunE: gitChangesConfirmPreRunHandler,
+		RunE:    scaffoldWasmHandler,
+	}
+
+	return c
+}
+
+func scaffoldWasmHandler(cmd *cobra.Command, _ []string) error {
+	session := cliui.New(cliui.StartSpinnerWithText(statusWasmInit))
+	defer session.End()
+
+	var (
+		appPath = "."
```

Comment:
It would great to be able to specify the app path with flag, or command argument. Otherwise if the intention is to always run the init command inside the app directory then it might be better to define a package constant:

```go
const defaultAppPath = "."
```

The default app path could be "." when a path is not explicitly setted using a command flag.